ci/infra/jobs/build_amis/dw-al2-hbase-ami.yml

```
@@ -0,0 +1,106 @@
+jobs:
+  - name: untested-dw-al2-hbase-ami
+    plan:
+      - get: hardened-ami-id
+        trigger: true
+        passed:
+          - dw-al2-test-and-promote-hardened-ami
+      - get: dw-al2-hbase-ami-config
+        trigger: true
+      - get: dw-al2-base-ami-template
+        trigger: false
+      - get: dw-al2-hardened-ami-config
+        trigger: false
+      - .: (( inject meta.plan.generate-manifest ))
+        config:
+          params:
+            AMI_NAME: untested-dw-al2-hbase-ami
+            SOURCE_AMI_NAME: dw-al2-hardened-ami-*
+            SOURCE_AMI_OWNER: ((aws_account.management))
+            PROVISION_SCRIPT_KEYS: '["dw-al2-hbase-ami-config/dw-al2-hbase-ami/dw-hbase-ami-install.sh"]'
+            PROVISIONER_TYPE_FILE_SOURCE: "dw-al2-hbase-ami-config/dw-al2-hbase-ami/"
+      - .: (( inject meta.plan.packer-bootstrap ))
+        input_mapping:
+          source_config: dw-al2-hardened-ami-config
+          source_template: dw-al2-base-ami-template
+      - .: (( inject meta.plan.build-ami ))
+        config:
+          inputs:
+            - name: dw-al2-hbase-ami-config
+      - put: untested-dw-al2-hbase-ami
+        params:
+          file: built-ami/id
+
+  - name: validate-hbase-ami
+    plan:
+      - get: untested-dw-al2-hbase-ami
+        trigger: true
+        passed:
+          - untested-dw-al2-hbase-ami
+      - get: analytical-dataset-generation
```

```suggestion
      - get: aws-internal-compute
```
Hbase should be in its own repo/pipeline....ew

---

this should be repeated everywhere else, and the resource created